pyanaconda/ui/gui/hubs/__init__.py

```diff
@@ -60,6 +60,7 @@ class Hub(GUIObject, common.Hub):
     """
```

Comment:
"*when a new Hub is created, it will first disable...and then it will enable...*"